bindata/v4.1.0/alerts/audit-errors.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: audit-errors
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+  - name: apiserver-audit
+    rules:
+    - alert: AuditLogError
+      annotations:
+        summary: |-
+          Incident response teams rely on the audit logs to detect malicious activity.
```

Comment:
would expect a more direct description of the error first. This paragraph is good, but it's more like background info.

Reply:
@sttts I updated the text, what do you think?